modules/cratedb/testcontainers/cratedb/__init__.py

```diff
@@ -0,0 +1,158 @@
+import os
+import typing as t
+from urllib.parse import quote
+
+from testcontainers.core.container import DockerContainer
+from testcontainers.core.exceptions import ContainerStartException
+from testcontainers.core.utils import raise_for_deprecated_parameter
+from testcontainers.core.wait_strategies import HttpWaitStrategy
+
+
+# DockerSkippingContainer, KeepaliveContainer,
+class CrateDBContainer(DockerContainer):
+    """
+    CrateDB database container.
+
+    Example:
+
+        The example spins up a CrateDB database and connects to it using
+        SQLAlchemy and its Python driver.
+
+        .. doctest::
+
+            >>> from testcontainers import cratedb import CrateDBContainer
+            >>> import sqlalchemy
+
+            >>> cratedb_container =
+            >>> with CrateDBContainer("crate:6.0") as cratedb:
+            ...     engine = sqlalchemy.create_engine(cratedb.get_connection_url())
+            ...     with engine.begin() as connection:
+            ...         result = connection.execute(sqlalchemy.text("select version()"))
+            ...         version, = result.fetchone()
+            >>> version
+            'CrateDB 6.0.2..'
+    """
+
+    CMD_OPTS: t.ClassVar[dict[str, str]] = {
+        "discovery.type": "single-node",
+        "node.attr.storage": "hot",
+        "path.repo": "/tmp/snapshots",
```

Comment:
Those two settings are probably not suitable for a minimal/default configuration.

Reply:
Gotcha